README.md
# qhadmin
Station group management system


public/news
[{
        "uid": 0,
        "title": "郎朗独奏音乐会",
        "date": "2017-02-20 / 13:30",
        "url": "javascript:void(0);"
    },
    {
        "uid": 1,
        "title": "中国美术学院工会工作奖励和配套资助办法",
        "date": "2017-02-20 / 13:30",
        "url": "javascript:void(0);"
    },
    {
        "uid": 2,
        "title": "中国美术学院举行纪念建团95周年活动",
        "date": "2017-02-20 / 13:30",
        "url": "javascript:void(0);"
    },
    {
        "uid": 3,
        "title": "“治水最前线”中国美术学院下乡创作实践展开幕,中国美术学院下乡创作实践展开幕",
        "date": "2017-02-20 / 13:30",
        "url": "javascript:void(0);"
    },
    {
        "uid": 4,
        "title": "「民族翰骨——潘天寿诞辰120年纪念大展」在中国美术馆开幕",
        "date": "2017-02-20 / 13:30",
        "url": "javascript:void(0);"
    },
    {
        "uid": 5,
        "title": "纪念潘天寿诞辰120周年座谈会举行 刘延东出席并讲话",
        "date": "2017-02-20 / 13:30",
        "url": "javascript:void(0);"
    },
    {
        "uid": 6,
        "title": "中国美术学院第八届教代会、第十六届工代会第三次会议隆重举行",
        "date": "2017-02-20 / 13:30",
        "url": "javascript:void(0);"
    },
    {
        "uid": 7,
        "title": "中国美术学院举行纪念建团95周年活动",
        "date": "2017-02-20 / 13:30",
        "url": "javascript:void(0);"
    }
]

js/public_bianji.js
/*
 * @Author: Alex chenzeyongjsj@163.com 
 * @Date: 2018-01-08 10:45:11 
 * @Last Modified by: Alex chenzeyongjsj@163.com
 * @Last Modified time: 2018-01-10 18:52:21
 */
window.onload = function () {
	(function () {
		var header = $('#header');
		header.load('tpl/header_bianji.html', function () {});
		var footer = $('#footer');
		footer.load('tpl/footer.html', function () {
			var open_pop = $('.tools-open-pop');
			var pop_box = $('.tools-pop');
			var pop_con = pop_box.find('.pop-container');
			var close_pop = pop_box.find('.icon-close');
			var close_pop_bg = pop_box.find('.pop-control');
			//打开工具箱
			open_pop.click(function () {
				pop_box.animate({
					width: '1%',
					height: '100%'
				}, 150, function () {
					pop_box.animate({
						width: '100%'
					}, 150, function () {
						pop_con.animate({
							opacity: 1
						}, 150);
					});
				});
			});
			//关闭工具箱
			close_pop.click(function () {
				pop_close();
			});
			close_pop_bg.click(function () {
				pop_close();
			});

			function pop_close() {
				pop_con.animate({
					opacity: 0
				}, 150, function () {
					pop_box.animate({
						width: '1%',
						height: '100%'
					}, 150, function () {
						pop_box.animate({
							width: 0,
							height: 0
						}, 150);
					});
				});
			}
		})
	})();
}

js/public_leader.js
/*
 * @Author: Alex chenzeyongjsj@163.com 
 * @Date: 2018-01-08 10:45:11 
 * @Last Modified by: Alex chenzeyongjsj@163.com
 * @Last Modified time: 2018-01-10 18:57:06
 */
window.onload = function () {
	(function () {
		var header = $('#header');
		header.load('tpl/header_leader.html', function () {});
		var footer = $('#footer');
		footer.load('tpl/footer.html', function () {
			var open_pop = $('.tools-open-pop');
			var pop_box = $('.tools-pop');
			var pop_con = pop_box.find('.pop-container');
			var close_pop = pop_box.find('.icon-close');
			var close_pop_bg = pop_box.find('.pop-control');
			//打开工具箱
			open_pop.click(function () {
				pop_box.animate({
					width: '1%',
					height: '100%'
				}, 150, function () {
					pop_box.animate({
						width: '100%'
					}, 150, function () {
						pop_con.animate({
							opacity: 1
						}, 150);
					});
				});
			});
			//关闭工具箱
			close_pop.click(function () {
				pop_close();
			});
			close_pop_bg.click(function () {
				pop_close();
			});

			function pop_close() {
				pop_con.animate({
					opacity: 0
				}, 150, function () {
					pop_box.animate({
						width: '1%',
						height: '100%'
					}, 150, function () {
						pop_box.animate({
							width: 0,
							height: 0
						}, 150);
					});
				});
			}
		})
	})();
}

js/public_xitong.js
/*
 * @Author: Alex chenzeyongjsj@163.com 
 * @Date: 2018-01-08 10:45:11 
 * @Last Modified by: Alex chenzeyongjsj@163.com
 * @Last Modified time: 2018-01-10 20:19:39
 */
window.onload = function () {
	(function () {
		var header = $('#header');
		header.load('tpl/header_xitong.html', function () {});
		var footer = $('#footer');
		footer.load('tpl/footer.html', function () {
			var open_pop = $('.tools-open-pop');
			var pop_box = $('.tools-pop');
			var pop_con = pop_box.find('.pop-container');
			var close_pop = pop_box.find('.icon-close');
			var close_pop_bg = pop_box.find('.pop-control');
			//打开工具箱
			open_pop.click(function () {
				pop_box.animate({
					width: '1%',
					height: '100%'
				}, 150, function () {
					pop_box.animate({
						width: '100%'
					}, 150, function () {
						pop_con.animate({
							opacity: 1
						}, 150);
					});
				});
			});
			//关闭工具箱
			close_pop.click(function () {
				pop_close();
			});
			close_pop_bg.click(function () {
				pop_close();
			});

			function pop_close() {
				pop_con.animate({
					opacity: 0
				}, 150, function () {
					pop_box.animate({
						width: '1%',
						height: '100%'
					}, 150, function () {
						pop_box.animate({
							width: 0,
							height: 0
						}, 150);
					});
				});
			}
		})
	})();
}

js/public_zhuzhan.js
/*
 * @Author: Alex chenzeyongjsj@163.com 
 * @Date: 2018-01-08 10:45:11 
 * @Last Modified by: Alex chenzeyongjsj@163.com
 * @Last Modified time: 2018-01-10 19:57:03
 */
window.onload = function () {
	(function () {
		var header = $('#header');
		header.load('tpl/header_zhuzhan.html', function () {});
		var footer = $('#footer');
		footer.load('tpl/footer.html', function () {
			var open_pop = $('.tools-open-pop');
			var pop_box = $('.tools-pop');
			var pop_con = pop_box.find('.pop-container');
			var close_pop = pop_box.find('.icon-close');
			var close_pop_bg = pop_box.find('.pop-control');
			//打开工具箱
			open_pop.click(function () {
				pop_box.animate({
					width: '1%',
					height: '100%'
				}, 150, function () {
					pop_box.animate({
						width: '100%'
					}, 150, function () {
						pop_con.animate({
							opacity: 1
						}, 150);
					});
				});
			});
			//关闭工具箱
			close_pop.click(function () {
				pop_close();
			});
			close_pop_bg.click(function () {
				pop_close();
			});

			function pop_close() {
				pop_con.animate({
					opacity: 0
				}, 150, function () {
					pop_box.animate({
						width: '1%',
						height: '100%'
					}, 150, function () {
						pop_box.animate({
							width: 0,
							height: 0
						}, 150);
					});
				});
			}
		})
	})();
}

js/public.js
/*
 * @Author: Alex chenzeyongjsj@163.com 
 * @Date: 2018-01-08 10:45:11 
 * @Last Modified by: alex (chenzeyongjsj@163.com)
 * @Last Modified time: 2018-01-22 14:19:16
 */
window.onload = function () {
	(function () {
		var header = $('#header');
		header.load('tpl/header.html', function () {
			$('.header-name').text(localStorage.getItem("headerName"));
			$('.header-url').attr('href', localStorage.getItem("headerUrl"));
			$('.notice-list').addClass(localStorage.getItem("infoHidden"));
			$('.xitong').addClass(localStorage.getItem("setHidden"));
			//退出登录
			$('.sign-out').click(function () {
				localStorage.clear();
			});
		});
		var footer = $('#footer');
		footer.load('tpl/footer.html', function () {
			var open_pop = $('.tools-open-pop');
			var pop_box = $('.tools-pop');
			var pop_con = pop_box.find('.pop-container');
			var close_pop = pop_box.find('.icon-close');
			var close_pop_bg = pop_box.find('.pop-control');
			//打开工具箱
			open_pop.click(function () {
				pop_box.animate({
					width: '1%',
					height: '100%'
				}, 150, function () {
					pop_box.animate({
						width: '100%'
					}, 150, function () {
						pop_con.animate({
							opacity: 1
						}, 150);
					});
				});
			});
			//关闭工具箱
			close_pop.click(function () {
				pop_close();
			});
			close_pop_bg.click(function () {
				pop_close();
			});

			function pop_close() {
				pop_con.animate({
					opacity: 0
				}, 150, function () {
					pop_box.animate({
						width: '1%',
						height: '100%'
					}, 150, function () {
						pop_box.animate({
							width: 0,
							height: 0
						}, 150);
					});
				});
			}
			/* 缓存dialog */
			var chche_box = $('.cache-dialog');
			$('.cache-btn').click(function () {
				chche_box.addClass('cache-dialog-open');
			});
			$('.dialog-close').click(function () {
				chche_box.removeClass('cache-dialog-open');
			});
		})
		//系统管理员
		if ($('#sideBar')) {
			var sideBar = $('#sideBar');
			sideBar.load('tpl/side_bar.html', function () {
				var li = $('.side-ul>li');
				var liClick = $('.side-ul>li>a');
				var _li = li.find('.side-click');
				var ol_li = li.find('li');
				//获取菜单的定位信息
				var liIndex = parseInt(window.sessionStorage.getItem("liIndex"));
				var aIndex = parseInt(window.sessionStorage.getItem("aIndex"));
				(function () { //初始化给菜单栏定位
					$(li[liIndex]).addClass('li-active');
					$(li[liIndex]).children('ol').children('li').eq(aIndex).addClass('side-active');
				})();
				liClick.click(function () {
					li = $(this).parent('li');
					if (li.hasClass('li-active')) {
						li.removeClass('li-active');
						$($('.side-ul>li')[liIndex]).addClass('li-active');
					} else {
						li.addClass('li-active').siblings('li').removeClass('li-active');
					}
				});
				_li.click(function () {
					liIndex = $(this).parents('ol').parent('li').index();
					aIndex = $(this).parent().index();
					//存储菜单的定位信息
					window.sessionStorage.setItem("liIndex", liIndex);
					window.sessionStorage.setItem("aIndex", aIndex);
				});
				/* 
				系统信息弹框
				*/
				$('.system-dialog').click(function () {
					$('.system-dialog-box').addClass('system-dialog-box-show');
				});
				$('.icon-close').click(function () {
					$('.system-dialog-box').removeClass('system-dialog-box-show');
				});
			});
		}
	})();
}